modules/database/src/ioc/misc/dlload.c

```diff
@@ -13,6 +13,7 @@
 IOCSH_STATIC_FUNC void dlload(const char* name)
 {
     if (!epicsLoadLibrary(name)) {
+        iocshSetError(-1);
```

Comment:
I think this call should be moved down into `dlloadCallFunc()`.  Which will mean changing the return type of `dlload()`.